hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -779,24 +879,61 @@ func (r *HostedClusterReconciler) reconcileIgnitionServer(ctx context.Context, h
 							Effect: corev1.TaintEffectNoSchedule,
 						},
 					},
+					Volumes: []corev1.Volume{
+						{
+							Name: "serving-cert",
+							VolumeSource: corev1.VolumeSource{
+								Secret: &corev1.SecretVolumeSource{
+									SecretName: servingCertSecret.Name,
```

Comment:
don't we need to set DefaultMode here?

Reply:
Default is 0644, should it be something else?

Reply:
I've seen 420 commonly used.

Reply:
Actually based on the comment they are equivalent
```
	// Optional: mode bits used to set permissions on created files by default.
	// Must be an octal value between 0000 and 0777 or a decimal value between 0 and 511.
	// YAML accepts both octal and decimal values, JSON requires decimal values
	// for mode bits. Defaults to 0644.
	// Directories within the path are not affected by this setting.
	// This might be in conflict with other options that affect the file
	// mode, like fsGroup, and the result can be other mode bits set.
	// +optional
```